src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -912,11 +912,17 @@ TxQ::apply(Application& app, OpenView& view,
                     LastLedgerSeq and MaybeTx::retriesRemaining.
                 */
                 auto const balance = (*sle)[sfBalance].xrp();
-                // Get the minimum possible reserve. If fees exceed
-                // this amount, the transaction can't be queued.
-                // TODO(?) Get potential owner count changes in
-                // calculateConsequences so this estimate can be
-                // more accurate
+                /* Get the minimum possible reserve. If fees exceed
+                   this amount, the transaction can't be queued.
+                   Considering that typical fees are several orders
+                   of magnitude than any current or expected future
```

Comment:
> several orders of magnitude **smaller** than

I guess?

Reply:
Yeah, I got distracted. It's pushed now.